src/net_processing.cpp

```diff
@@ -601,6 +604,9 @@ class PeerManagerImpl final : public PeerManager
     /** Number of preferable block download peers. */
     int nPreferredDownload GUARDED_BY(cs_main) = 0;
 
+    /** Height of highest fast announce block, updated by NewPoWValidBlock */
+    int m_height_of_highest_fast_announce GUARDED_BY(cs_main) = 0;
```

Comment:
Maybe `m_height_highest_cb_fast_announced`, inducing a unit is always better IMO.